### PR TITLE
Hide submit button using class instead of left-margin, makes it usable with CSP

### PR DESCRIPTION
**Pull request by mathiasertl**
Hide submit button using class instead of left-margin, makes it usable with CSP

## Description

Replace inline CSS styles with existing bootstrap CSS class to hide the submit button.

I'm still a bit unhappy with the empty paragraph as hack. Without it, the table and the submit button row has no space between them. I can think of two solutions:

* Upgrade to bootstrap4 (which has appropriate margin classes) and add them to the table.
* Add a static CSS file with a custom class that adds that same margin.

If any of the two seem better to you, please let me know, I'm happy to amend the PR.

## Motivation and Context

This makes the default templates usable with a restrictive [Content-Security Policy](https://en.wikipedia.org/wiki/Content_Security_Policy).

## How Has This Been Tested?

I could not find any unittests for the UI. If there are, please let me know.

I have manually tested the views, and also tried the enter button of course (that this submit button seems to facilitate).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by claudep** [comment]
Note that the patch to move to Bootstrap 4 (#379) is only waiting for a final review.

**Comment by mathiasertl** [comment]
Aha! Excellent. Then I propose we wait wait for that PR and I will adapt mine then. 

In the meantime anything else I can do to improve the PR, or is it otherwise fine?

**Comment by moggers87** [comment]
> Note that the patch to move to Bootstrap 4 (#379) is only waiting for a final review.

noted

File: two_factor/templates/two_factor/_wizard_forms.html
Comment (moggers87) [inline two_factor/templates/two_factor/_wizard_forms.html:6]: If there needs to be extra space, then you need to add margin to the table. I think Boostrap has a class for that.

**Review by moggers87 — CHANGES_REQUESTED** [review]
If you could update your branch with the latest commits from master as well please.

File: two_factor/templates/two_factor/_wizard_forms.html
Comment (mathiasertl) [reply two_factor/templates/two_factor/_wizard_forms.html:6]: Bootstrap 4 has, Bootstrap 3 didn't ;-), that's why I added the paragraph-hack. Now updating the branch.

**Comment by mathiasertl** [comment]
I have updated the branch to use bs4 classes, and updated the changelog as well.

@claudep is there a reason the example project overrides some of the templates from the app? They seem like unnecessary code duplication to me. If you agree, I can make a separate PR for that.

**Comment by claudep** [comment]
I'm not the author of the example files (and don't care much about them, frankly).

**Comment by mathiasertl** [comment]
Alright. Then I propose we merge this as is.

**Review by moggers87 — APPROVED** [review]

**Pull request merged**
